udata_front/theme/gouvfr/assets/js/api/organizations.ts

```diff
@@ -0,0 +1,20 @@
+import { toValue } from "vue";
+import { api } from "../plugins/api";
+import type { Organization } from "../types";
+
+type UploadLogoResponse = {
+  image: string;
+  success: boolean;
+};
+
+export function createOrganization(organization: Organization) {
```

Comment:
use `MaybeRefOrGetter` if you want to allow ref as param or remove `toValue` if ref aren't allowed